examples/ASCollectionView/Sample/ViewController.m

```diff
@@ -70,8 +70,8 @@ - (void)viewDidLoad
     {
       NSLog(@"ViewController is not nil");
       strongSelf->_data = [[NSArray alloc] init];
-      [strongSelf->_collectionView performBatchUpdates:^{
-        [strongSelf->_collectionView insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
+      [strongSelf->_collectionNode performBatchUpdates:^{
+        [strongSelf->_collectionNode insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
       } completion:nil];
       NSLog(@"ViewController finished updating collectionView");
```

Comment:
Actually, if we keep using 2s it will cause crash at L115 ASCollectionLayout, because poping VC and layout happen at same time; So I keep 4. 
- (CGSize)collectionViewContentSize
{
  ...
  ASDisplayNodeAssertNotNil(_layout, @"Collection layout state should not be nil at this point")
...

Let me know if it's potential bug and how to fix it.  Thanks.